src/main/java/uk/gov/hmcts/reform/rse/idam/simulator/service/token/OpenIdConfigService.java

```diff
@@ -81,7 +81,6 @@ public OpenIdConfig getOpenIdConfig(String baseUrl, String issuer) {
                 "PS256"
             ))
             .rcsResponseEncryptionEncValuesSupported(Arrays.asList("A256GCM", "A128CBC-HS256", "A256CBC-HS512"))
-            .issuer(issuer)
```

Comment:
`issuer` was then being overridden again a few lines down with the `baseUrl` value